Modules/Relationships/test/RelationshipTemplates.API.Tests.Performance.EndToEnd/src/get-relationshipTemplate-by-id.test.ts

```diff
@@ -0,0 +1,92 @@
+import http from 'k6/http';
+import { Options } from 'k6/options';
+import { describe, expect } from "https://jslib.k6.io/k6chaijs/4.3.4.2/index.js";
+
+const host = __ENV.HOST;
+const apiEndpoint = host+'/api/v1/';
+
+interface Data {
+    authToken: string,
+    relationshipTemplateId: string
+}
+
+function translateSize() {
```

Comment:
Now that I see the parameterless function I think that we should call it `size()`